apis/externalsecrets/v1alpha1/secretstore_onepassword_types.go

```diff
@@ -31,7 +31,10 @@ type OnePasswordAuthSecretRef struct {
 
 // OnePasswordProvider configures a store to sync secrets using the 1Password Secret Manager provider.
 type OnePasswordProvider struct {
-	Auth        *OnePasswordAuth `json:"auth"`
-	ConnectHost string           `json:"connectHost"`
-	Vaults      []string         `json:"vaults"`
+	// Auth defines the information necessary to authenticate against OnePassword Connect Server
+	Auth *OnePasswordAuth `json:"auth"`
+	// ConnectHost defines the OnePassword Connect Server to connect to
+	ConnectHost string `json:"connectHost"`
+	// Vaults defines which OnePassword vaults to search in which order
+	Vaults map[string]int `json:"vaults"`
```

Comment:
These comments make their way in to the generated CRDs and help kubectl explain work better.